### ISSUE TITLE
Use JSHint to ensure code quality and make code compressable

### ISSUE THREAD

**Issue by LeJared** [thebird/Swipe#248]
I'd like to suggest using http://www.jshint.com/ to ensure code quality and make code robust for using js compressor tools.

**Comment by tobegit3hub** [comment]
+1

**Comment by lyfeyaj** [comment]
@tobegit3hub Hi, I maintain a independent version of this project, and this feature you wanted has already been added. See <https://github.com/lyfeyaj/swipe> for details. Hope that can help you.

**Comment by tobegit3hub** [comment]
Thanks @lyfeyaj :+1: Can you send the pull-request for this repo?

**Comment by lyfeyaj** [comment]
@tobegit3hub
Yes, I can send a pull request, but this repo seems to be abandoned by the author. For there is no updates in nearly two years. see #460 for discussion

**Comment by tobegit3hub** [comment]
Thanks for your contribution :smile:  @lyfeyaj
